test/api/test_history.py

```diff
@@ -0,0 +1,66 @@
+"""Caching and History API test for KAOS."""
+
+import json
+
+import context
+from  kaos.models import DB, ResponseHistory
```

Comment:
There are two spaces after `from`.

OMG, Zeyad, get it right and stop ruining my beautiful code base.

Reply:
God dam it, I have brought shame on myself and my family!